nixos/modules/services/networking/v2raya.nix

```
@@ -0,0 +1,32 @@
+{ config, pkgs, lib, ... }:
+
+{
+  options = {
+    services.v2raya = {
+      enable = lib.options.mkEnableOption (lib.mdDoc "the v2rayA service");
+    };
+  };
+
+  config = lib.mkIf config.services.v2raya.enable {
+    systemd.services.v2raya = {
+      unitConfig = {
+        Description = "v2rayA service";
+        Documentation = "https://github.com/v2rayA/v2rayA/wiki";
+        After = [ "network.target" "nss-lookup.target" "iptables.service" "ip6tables.service" ];
+        Wants = [ "network.target" ];
+      };
+
+      serviceConfig = {
+        User = "root";
+        ExecStart = "${pkgs.v2raya}/bin/v2rayA --log-disable-timestamp";
+        LimitNPROC = 500;
```

```suggestion
        Environment = [ "V2RAYA_LOG_FILE=/var/log/v2raya/v2raya.log" ];
        LimitNPROC = 500;
```

https://github.com/v2rayA/v2rayA/blob/feat_v5/install/universal/v2raya.service#L13